pylint/checkers/imports.py

```diff
@@ -444,29 +442,11 @@ def __init__(self, linter: Optional["PyLinter"] = None) -> None:
         self._site_packages = self._compute_site_packages()
 
     @staticmethod
-    def _compute_site_packages():
-        def _normalized_path(path):
+    def _compute_site_packages() -> Set[str]:
+        def _normalized_path(path: str) -> str:
```

Comment:
Another question: Where is it still used? I wanted to check if `STD_LIB_DIRS` is actual the correct replacement but couldn't find any code that uses `self._site_packges`.

Reply:
Yep, it looks unused. `pylint` used to rely on `EXT_LIB_DIRS`, until these special cases were added in 6e7037479b79b28ed8db4c74df7bff193f4574c4. Then the use of `_site_packages` was removed in 9812c5a49ae430945e9ad5dde7ec523904498805. I think this can just be removed without a replacement.